packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -593,6 +593,12 @@ export interface DatabaseInstanceNewProps {
    * @default - None
    */
   readonly s3ExportBuckets?: s3.IBucket[];
+  /**
+   * Indicates whether the DB instance is an internet-facing instance
+   *
+   * @default - `true` if the instance is deployed in a `PUBLIC` subnet, `false` otherwise.
```

Comment:
No need for the period here:

```suggestion
   * @default - `true` if the instance is deployed in a `PUBLIC` subnet, `false` otherwise
```